torch/autograd/gradcheck.py

```diff
@@ -277,7 +277,8 @@ def gradcheck(
             identical inputs through the differentiation, the results must either match
             exactly (default, 0.0) or be within this tolerance.
         check_undefined_grad (bool, options): if True, check if undefined output grads
-            are supported and treated as zeros
+            are supported and treated as zeros. Defaults to ``True`` for ``Tensor`` outputs
```

Comment:
nit: should this say inputs? Or should the check below check the outputs? I think the two should be consistent.

Reply:
Right, done!